internal/cli/dialog/proposals_show_lineage.go

```diff
@@ -0,0 +1,48 @@
+package dialog
+
+import (
+	"fmt"
+
+	"github.com/git-town/git-town/v22/internal/cli/dialog/dialogcomponents"
+	"github.com/git-town/git-town/v22/internal/cli/dialog/dialogcomponents/list"
+	"github.com/git-town/git-town/v22/internal/cli/dialog/dialogdomain"
+	"github.com/git-town/git-town/v22/internal/forge/forgedomain"
+	"github.com/git-town/git-town/v22/internal/messages"
+	. "github.com/git-town/git-town/v22/pkg/prelude"
+)
+
+const (
+	proposalsShowLineageTitle = `Proposals Show Lineage`
+	proposalsShowLineageHelp  = `
+	How should Git Town share stack lineage in proposals?
+`
+)
+
+func ProposalsShowLineage(args Args[forgedomain.ProposalsShowLineage]) (Option[forgedomain.ProposalsShowLineage], dialogdomain.Exit, error) {
+	entries := list.Entries[Option[forgedomain.ProposalsShowLineage]]{}
+	if global, hasGlobal := args.Global.Get(); hasGlobal {
+		entries = append(entries, list.Entry[Option[forgedomain.ProposalsShowLineage]]{
+			Data: None[forgedomain.ProposalsShowLineage](),
+			Text: fmt.Sprintf(messages.DialogUseGlobalValue, global),
+		})
+	}
+
+	entries = append(entries, list.Entries[Option[forgedomain.ProposalsShowLineage]]{
+		{
+			Data: Some(forgedomain.ProposalsShowLineageNone),
+			Text: "stack lineage is not displayed in proposals",
```

Comment:
Let's use active voice here.



```suggestion
			Text: "no stack lineage in proposals",
```